web/json_value_test.go

```diff
@@ -88,6 +88,46 @@ func TestConvertJSONAttributeValue(t *testing.T) {
 			valueJSON: `[true, 0, "true"]`,
 			wantValue: []bool{true, false, true},
 		},
+		"unix": {
+			attribute: &framework.AttributeConfig{
+				ExternalId: "a",
+				Type:       framework.AttributeTypeDateTime,
+			},
+			valueJSON: `"1706041056"`,
+
+			opts:      &jsonOptions{dateTimeFormats: []DateTimeFormatWithTimeZone{{"sgnl-unix-ts", false}, {"2006-01-02", false}}},
+			wantValue: MustParseTime(t, "2024-01-23T20:17:36Z"),
+		},
+		"neg_unix": {
+			attribute: &framework.AttributeConfig{
+				ExternalId: "a",
+				Type:       framework.AttributeTypeDateTime,
+			},
+			valueJSON: `"-1706041056"`,
+
+			opts:      &jsonOptions{dateTimeFormats: []DateTimeFormatWithTimeZone{{"sgnl-unix-ts", false}, {"2006-01-02", false}}},
+			wantValue: MustParseTime(t, "1915-12-10T03:42:24Z"),
+		},
+		"unix_with_tz_offset": {
+			attribute: &framework.AttributeConfig{
+				ExternalId: "a",
+				Type:       framework.AttributeTypeDateTime,
+			},
+			valueJSON: `"1706041056"`,
+
+			opts:      &jsonOptions{dateTimeFormats: []DateTimeFormatWithTimeZone{{"sgnl-unix-ts", true}, {"2006-01-02", false}}, localTimeZoneOffset: 60 * 60},
+			wantValue: MustParseTime(t, "2024-01-23T20:17:36Z"),
```

Comment:
Shouldn't we be expecting something different here after applying the local timezone offset?